server/views/referrals/new/additionalInformation/show.njk

```diff
@@ -64,14 +64,27 @@
           value: formValues.formattedAdditionalInformation or referral.additionalInformation,
           errorMessage: errors.messages.additionalInformation
         }) }}
-
-        {{ govukButton({
+        <div class="govuk-button-group">
```

Comment:
The first `form` tag is outside of `<div class="govuk-button-group">` and the corresponding closing tag is inside it so the html is invalid.

As discussed, if you were using 2 forms but wanted the buttons together, you need to put an `id` on the `form` and then specify that id with the `for` attribute on the button.